Tests flake: uncompleted tests due to panic: runtime error: index out of range

as seen in logs:

https://storage.googleapis.com/minikube-builds/logs/11106/05abe4a/Docker_Linux_containerd.txt:
```
...
--- FAIL: TestMultiNode (795.63s)
    --- FAIL: TestMultiNode/serial (795.63s)
        --- PASS: TestMultiNode/serial/FreshStart2Nodes (168.74s)
        --- FAIL: TestMultiNode/serial/DeployApp2Nodes (626.89s)
panic: runtime error: index out of range [1] with length 1 [recovered]
	panic: runtime error: index out of range [1] with length 1

goroutine 1978 [running]:
testing.tRunner.func1.2(0x1f24e60, 0xc0003d00d8)
	/usr/local/go/src/testing/testing.go:1144 +0x332
testing.tRunner.func1(0xc00052ea80)
	/usr/local/go/src/testing/testing.go:1147 +0x4b6
panic(0x1f24e60, 0xc0003d00d8)
	/usr/local/go/src/runtime/panic.go:965 +0x1b9
k8s.io/minikube/test/integration.validateDeployAppToMultiNode(0x244deb0, 0xc001322600, 0xc00052ea80, 0xc00148c1a0, 0x1f)
	/mnt/disks/sdb/jenkins/go/src/k8s.io/minikube/test/integration/multinode_test.go:413 +0x1474
k8s.io/minikube/test/integration.TestMultiNode.func1.1(0xc00052ea80)
	/mnt/disks/sdb/jenkins/go/src/k8s.io/minikube/test/integration/multinode_test.go:65 +0xa8
testing.tRunner(0xc00052ea80, 0xc000656bc0)
	/usr/local/go/src/testing/testing.go:1194 +0xef
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:1239 +0x2b3
...
```
and
https://storage.googleapis.com/minikube-builds/logs/11106/05abe4a/Docker_Linux_crio.txt:
```
...
--- FAIL: TestMultiNode (753.96s)
    --- FAIL: TestMultiNode/serial (753.96s)
        --- PASS: TestMultiNode/serial/FreshStart2Nodes (129.06s)
        --- FAIL: TestMultiNode/serial/DeployApp2Nodes (624.90s)
panic: runtime error: index out of range [1] with length 1 [recovered]
	panic: runtime error: index out of range [1] with length 1

goroutine 1991 [running]:
testing.tRunner.func1.2(0x1f24e60, 0xc001692690)
	/usr/local/go/src/testing/testing.go:1144 +0x332
testing.tRunner.func1(0xc000e54000)
	/usr/local/go/src/testing/testing.go:1147 +0x4b6
panic(0x1f24e60, 0xc001692690)
	/usr/local/go/src/runtime/panic.go:965 +0x1b9
k8s.io/minikube/test/integration.validateDeployAppToMultiNode(0x244deb0, 0xc00147f320, 0xc000e54000, 0xc000396d00, 0x1f)
	/mnt/disks/sdb/jenkins/go/src/k8s.io/minikube/test/integration/multinode_test.go:413 +0x1474
k8s.io/minikube/test/integration.TestMultiNode.func1.1(0xc000e54000)
	/mnt/disks/sdb/jenkins/go/src/k8s.io/minikube/test/integration/multinode_test.go:65 +0xa8
testing.tRunner(0xc000e54000, 0xc000eb2300)
	/usr/local/go/src/testing/testing.go:1194 +0xef
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:1239 +0x2b3
...
```
/assign

--- a/test/integration/multinode_test.go
+++ b/test/integration/multinode_test.go
@@ -419,8 +419,7 @@ func validateDeployAppToMultiNode(ctx context.Context, t *testing.T, profile str
 	podIPs := strings.Split(strings.Trim(rr.Stdout.String(), "'"), " ")
 	if len(podIPs) != 2 {
 		t.Errorf("expected 2 Pod IPs but got %d", len(podIPs))
-	}
-	if podIPs[0] == podIPs[1] {
+	} else if podIPs[0] == podIPs[1] {
 		t.Errorf("expected 2 different pod IPs but got %s and %s", podIPs[0], podIPs[0])
 	}
 

